Controller/MediaController.php

```diff
@@ -44,27 +45,28 @@ public function getMedia($id)
     /**
      * @throws NotFoundHttpException
      *
-     * @param string $id
-     * @param string $format
+     * @param Request $request
+     * @param string  $id
+     * @param string  $format
      *
      * @return Response
      */
-    public function downloadAction($id, $format = 'reference')
+    public function downloadAction(Request $request, $id, $format = 'reference')
```

Comment:
This is a BC-break, which means there should be something in the `UPGRADE` guide at least, and at most, a PR should be made on the stable branch to simulate both signatures, and deprecate the old one (that would help with migrating calls, but not with migrating child classes).